contents/ui/CompactRepresentation.qml

```diff
@@ -19,18 +19,30 @@ import QtQuick 2.0
 import QtQuick.Layouts 1.1
 import org.kde.plasma.components 2.0 as PlasmaComponents
 import org.kde.plasma.calendar 2.0 as PlasmaCalendar
+import ApplicationLauncher 1.0
+
+Application {
+  id: launch  
+  appName: PlasmaCalendar.Calendar.calendarBackend;
+}
+Timer {
+	interval: 500
```

Comment:
I don't think this is the best solution. Consider this, it is a timer triggered every 500 ms to update a value that changes once a week, i.e. twice per second, 2x60=120 per minute, 120*60=7200 per hour ... 1209600 per week.

I would go for a different solution. calendarBackend exposes to this C++ API: https://api.kde.org/frameworks/plasma-framework/html/calendar_8h_source.html . currentWeek is not a QPROPERTY, it is just a QML invokable, so we can't rely on something like a currentWeekChanged signal. However, there are other signals that could help us to update the week number in a more effective way.

Reply:
Have you fixed that?

Reply:
Nope

Reply:
Why?

Reply:
I got no time for this, too many life changes since #4 was filled. I left this PR open with hopes to get an update from the author, and that did not happen. If you manage to propose a better patch, please do so.

Reply:
Hi @anselmolsm. First of all, thank you for this plasmoid. I just started using it and I find it very handy on my work laptop.

It would be very nice if this bug was fixed. How about switching the delay to 1 update/minute? Surely this is nothing on modern computers.

I think it would be better to merge this and have the plasmoid update in a suboptimal way than let it show last week's number on a Monday afternoon :)